modules/Rock.cmake

```diff
@@ -1471,6 +1522,7 @@ function(rock_add_test TARGET_NAME __rock_test_parameters)
     add_test(NAME test-${TARGET_NAME}-cxx
              COMMAND ${EXECUTABLE_OUTPUT_PATH}/${TARGET_NAME}
              ${__rock_test_parameters})
+    set(TEST_TARGET_NAME "test-${TARGET_NAME}-cxx" PARENT_SCOPE)
```

Comment:
Nope. Don't do that.

rock_add_test generates a target called `test-${TARGET_NAME}-cxx`. You are allowed to rely on that in `rock_cxx_coverage_report`